diff -Nru vmanpg-1.1/bdf2h.c vmanpg-1.1.pl/bdf2h.c
--- vmanpg-1.1/bdf2h.c	Wed Jul 26 13:43:33 2000
+++ vmanpg-1.1.pl/bdf2h.c	Wed Jan 29 10:09:18 2003
@@ -31,16 +31,16 @@
 int fw,fh,fox,foy,width,height,x,y,wcount;
 int maxy=-999;
 int ofs=0;
-int ofstbl[128];
+int ofstbl[256];
 char fontdesc[256]="unnamed";
 
 if(argc==2) strcpy(fontdesc,argv[1]);
 
-/* we explicitly specify `signed' because it needs to be.
+/* we explicitly specify `unsigned' because it needs to be.
  * I guess it doesn't matter 'cos it'll always be compiled on
  * a compiler where signed is the default, but what the hell.
  */
-printf("static signed char font%sdat[]={\n\n",fontdesc);
+printf("static unsigned char font%sdat[]={\n\n",fontdesc);
 
 printf("/* data for each char is ox oy w h dwidth, then data */\n\n");
 
@@ -58,7 +58,7 @@
     }
   else if(strcmp(buf,"BITMAP\n")==0)
     {
-    if(c<32 || c>127) continue;
+    if(c<32 || c>255) continue;
     ofstbl[c]=ofs;
     printf("/* `%c' */\n",c);
     printf("%d,%d,%d,%d,%d,\n",x,y,width,height,w);
@@ -97,10 +97,10 @@
 
 printf("};\n\n");
 
-/* lookup table for each char (32..127) */
-printf("static int font%stbl[96]={",fontdesc);
+/* lookup table for each char (32..255) */
+printf("static int font%stbl[224]={",fontdesc);
 ofstbl[127]=0;		/* well, ok, no char 127 :-) */
-for(f=32;f<128;f++)
+for(f=32;f<256;f++)
   {
   if((f&7)==0) putchar('\n');
   printf("%d,",ofstbl[f]);
diff -Nru vmanpg-1.1/font.c vmanpg-1.1.pl/font.c
--- vmanpg-1.1/font.c	Wed Jul 26 13:43:40 2000
+++ vmanpg-1.1.pl/font.c	Wed Jan 29 10:09:18 2003
@@ -119,7 +119,7 @@
 
 
 
-int vmode_drawtext(int xpos,int ypos,int attr,char *str)
+int vmode_drawtext(int xpos,int ypos,int attr, unsigned char *str)
 {
 int f,c,x,y;
 int initxpos=xpos;
@@ -200,7 +200,7 @@
     xpos=X_MARGIN+(((xpos-X_MARGIN)/TABSIZE)+1)*TABSIZE;
     continue;
     }
-  if(c<32 || c>126) c='_';
+  if(c<32) c='_';
   ptr=fontdat+fonttbl[c-32];
   ox=*ptr++; oy=*ptr++;
   w =*ptr++; h =*ptr++;
diff -Nru vmanpg-1.1/font.h vmanpg-1.1.pl/font.h
--- vmanpg-1.1/font.h	Wed Jul 26 13:43:47 2000
+++ vmanpg-1.1.pl/font.h	Wed Jan 29 10:14:01 2003
@@ -8,7 +8,7 @@
 
 extern int invert;
 
-extern int vmode_drawtext(int xpos,int ypos,int attr,char *str);
+extern int vmode_drawtext(int xpos,int ypos,int attr,unsigned char *str);
 extern void vmode_start(void);
 extern void vmode_clear(void);
 extern void vmode_clearlines(int y1,int y2);
diff -Nru vmanpg-1.1/vmanpg.c vmanpg-1.1.pl/vmanpg.c
--- vmanpg-1.1/vmanpg.c	Wed Jul 26 13:42:45 2000
+++ vmanpg-1.1.pl/vmanpg.c	Wed Jan 29 10:09:18 2003
@@ -287,7 +287,7 @@
   vmode_refresh();
   
   key=getkey();
-  if((key>=32 && key<=126) || key==9)	/* allow tab too */
+  if((key>=32) || key==9)	/* allow tab too */
     {
     if(iptr<ibuf+sizeof(ibuf)-1)
       {
